### ISSUE TITLE
Logout is missing

### ISSUE THREAD

**Issue by PavelNiedoba** [netbirdio/netbird#1569]
**Describe the problem**
I need to logout securely. I means that private keys and server peer is deleted and it's not possible to login without going thorough authentication. 

A clear and concise description of what the problem is.
For security and testing (switching between users) I need to logout. If i do `netbird down` I can do later `netbird up` and I will be connected back without any interaction. Password is not required.

`netbird-gui` is broken on ubuntu. I can connect, then disconnect, but connect will stay disabled. I can close gui and then reconnect. Anybody ever tested this?

**To Reproduce**

do netbird up, netbird down and netbird up again. Password will not be needed second time.

**Expected behavior**
peer should be removed from server,
password should be asked second time

Vpn is security feature, which become useless it logout is not possible.  If somebody can steal laptop and reopen connections to protected resources is security hole.

**Are you using NetBird Cloud?**
never

Please specify whether you use NetBird Cloud or self-host NetBird's control plane.
self hosted
**NetBird version**
0.25.7

**Comment by lixmal** [comment]
Have you [enabled the enforce periodic user authentication feature](https://docs.netbird.io/how-to/enforce-periodic-user-authentication)?

**Comment by b100dian** [comment]
There is an /etc/netbird/config.json or /var/snap/netbird/common/config.json that can be moved away (deleted if you know what you're doing) to achieve this in my experience

**Comment by HBenzaoui** [comment]
Agreed! we definitely need a logout command, it would be awesome great to have it, I used to logout all the time in tailscale and change account just from the terminal, I hope someone will bring this feature for the awesome netbird

**Comment by TJ-coding** [comment]
When will this be implemented? This alone is making me consider switching to Cloudflare Tunnel

**Comment by tony1661** [comment]
Will this ever be implemented?

**Comment by Cristobal-M** [comment]
I have an issue that might be resolved with an automatic logout functionality in an **ephemeral peer** scenario.

I’m using ephemeral peers in a Kubernetes cluster, deployed to access internal services. When the pods are replaced, I get timeout errors. I have to either manually delete the peers in the NetBird dashboard or wait for them to expire.

**Comment by nazarewk** [comment]
> I’m using ephemeral peers in a Kubernetes cluster, deployed to access internal services. When the pods are replaced, I get timeout errors. I have to either manually delete the peers in the NetBird dashboard or wait for them to expire.

@Cristobal-M could you create a separate report and elaborate? 
- new Pods should have unique hostnames, so they shouldn't cause issues with name clashes and adding `-N` suffix
- if the traffic is still going towards Peers that went offline this is not expected and should be investigated further

**Comment by Cristobal-M** [comment]
> > I’m using ephemeral peers in a Kubernetes cluster, deployed to access internal services. When the pods are replaced, I get timeout errors. I have to either manually delete the peers in the NetBird dashboard or wait for them to expire.
> 
> [@Cristobal-M](https://github.com/Cristobal-M) could you create a separate report and elaborate?
> 
>     * new Pods should have unique hostnames, so they shouldn't cause issues with name clashes and adding `-N` suffix
> 
>     * if the traffic is still going towards Peers that went offline this is not expected and should be investigated further

Hi @nazarewk  I have filled an issue about this, It lloks like an issue with DNS Routing when one peer disconnects. I'm not trying to make requests to the peer, but another host with dns name in the peer's subnet.
[https://github.com/netbirdio/netbird/issues/3976](https://github.com/netbirdio/netbird/issues/3976)

**Comment by nazarewk** [comment]
FYI: We stumbled upon an actual performance-related use case for implementing this, so we will look into implementing a `netbird logout` (or something similar) in the near future.

**Comment by matanbaruch** [comment]
@PavelNiedoba 
Implemented
https://github.com/netbirdio/netbird/pull/4268

**Comment by saule1508** [comment]
So https://github.com/netbirdio/netbird/pull/4268 is not a logout feature, it was renamed to deregister.
The use case of the logout is very important for our users. Let's say you need to start an intervention on a system and you need netbird to access the server. Currently, when a user logs in it remains logged in for x hours. So it is very well possible than 5 minutes after he starts the intervention he will be logged out. 
A logout would allow to reset the login validity to something long

**Comment by nazarewk** [comment]
> A logout would allow to reset the login validity to something long

Sounds to me like you want a session extending feature like https://github.com/netbirdio/netbird/issues/4209 instead of `logout`.
I will be closing this one as implemented by #4268 , as of writing this message the command is now called `deregister`, but still has `logout` alias to it.
